src/components/transport_manager/include/transport_manager/usb/usb_aoa_adapter.h

```diff
@@ -44,8 +44,13 @@ namespace transport_adapter {
 
 class UsbAoaAdapter : public TransportAdapterImpl {
  public:
-  UsbAoaAdapter(resumption::LastState& last_state,
+  UsbAoaAdapter(resumption::LastStateWrapperPtr last_state_wrapper,
                 const TransportManagerSettings& settings);
+
+  DEPRECATED
+  UsbAoaAdapter(resumption::LastState&,
```

Comment:
Include proper function definition

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
Still no definition, see https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r372663000 for an example

Reply:
@jacobkeeler Please take a look https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r373481596